javalib/src/main/scala/java/util/concurrent/ConcurrentSkipListSet.scala

```diff
@@ -46,7 +46,8 @@ class ConcurrentSkipListSet[E] protected (ordering: Ordering[_ >: E], _comparato
     inner.isEmpty
 
   override def contains(o: Any): Boolean =
-    inner.contains(Box(o.asInstanceOf[E]))
+    if (o == null) false
+    else inner.contains(Box(o.asInstanceOf[E]))
```

Comment:
Why is this change needed? `Box` is supposed to be perfectly happy with a `null` value.